assets/src/layout/Header/index.tsx

```diff
@@ -191,7 +191,7 @@ const Header: React.FC = () => {
           </PrimaryContainer>
         </PrimaryHeader>
 
-        {isAuthenticated ? (
+        {isAuthenticated && user ? (
```

Comment:
@acodegirl does this logic make sense? "if authenticated and if user". If authenticated I would expect there to always be a user, so logically I'm not sure the right hand side of this operation should be required?

Reply:
@jonathaningram - Soon after authentication, its taking a bit of time to fetch pay user data. But meanwhile couple of graphql requests are sent to fetch service related info which were failing and sent again after receiving successful user information. This is the reason i added this check to ensure graphql requests are sent only after fetching user info. Please refer  the above screenshot attached by Peter.

Reply:
@acodegirl ok no problem thanks